### PR TITLE
Update skip level section

### PR DESCRIPTION
**Pull request by msennett22**
Updated skip level section with purpose/process

@nicksnyder I didn't commit to a specific cadence, but am thinking we should give people an option to schedule a meeting with you if they want to meet more frequently/sooner than what you propose

### CODE REVIEW

**Comment by nicksnyder** [comment]
Key points
* As our department continues to grow, I also want to regularly check in with teammates to keep a pulse on what is happening.
* These meetings are an opportunity to build deeper relationships, for me to listen, gather your feedback and insights, and discuss anything top of mind for you. 
* Happy to talk about anything that is top of your mind or answer any questions that you might have.
* These are intended to be casual conversations where we can chat informally
* If I send you an invite to chat but the timing is bad or you don't have anything you want to discuss, please decline.
* I don't expect status updates, that is not the intent of these

File: content/company/team/nick.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ If I am your new manager, I will setup a series of three extra 1-1s to learn mor
 
 A skip level 1-1 is between me and someone who I indirectly manage (e.g. the person reports to a person that I manage).
 
-If you would like to have a skip level 1-1 with me:
+These 1-1s are an opportunity to build deeper relationships, for me to listen, gather your feedback and insights, and discuss anything top of mind for you.
+
+Engineering Managers and Engineers will receive calendar invites from me throughout the year. If you want to meet more frequently:
 
 1. Find a time on my calendar that is open and send me a calendar invite.
 2. Include as much context as is relevant so that I can prepare for the 1-1 if necessary.
```
Comment (nicksnyder) [inline content/company/team/nick.md]: Reduce to please either put something directly on my calendar or reach out to Michal to help schedule

**Comment by msennett22** [comment]
Updated based on all your feedback @nicksnyder 

I am having difficulty linking to my bio correctly now that this new team.yml file was created. It keeps directing to the github team page.

File: content/company/team/nick.md [hunk 2]
```diff
@@ -18,10 +18,13 @@ If I am your new manager, I will setup a series of three extra 1-1s to learn mor
...
-If you would like to have a skip level 1-1 with me:
+As our department continues to grow, I want to regularly check in with our teammates to keep a pulse on what is happening.
 
-1. Find a time on my calendar that is open and send me a calendar invite.
-2. Include as much context as is relevant so that I can prepare for the 1-1 if necessary.
+These 1-1s are intended to be casual and informal conversations to build deeper relationships, for me to listen, gather your feedback and insights, and discuss anything top of mind for you. These are not intended for status updates.
+
+If you receive a calendar invite from me but you need to change the date/time or you prefer not to meet, please update the invite.
+
+If you want to meet more frequently, please schedule time on my calendar or reach out to [Michal](../../../data/team.yml#michal-sennett) to help schedule.
```
Comment (nicksnyder) [inline content/company/team/nick.md]:
```suggestion
If you want to meet more frequently, please schedule time on my calendar or reach out to [Michal](index.md#michal-sennett) to help schedule.
```

File: content/company/team/nick.md [hunk 3]
```diff
@@ -18,10 +18,13 @@ If I am your new manager, I will setup a series of three extra 1-1s to learn mor
...
 A skip level 1-1 is between me and someone who I indirectly manage (e.g. the person reports to a person that I manage).
 
-If you would like to have a skip level 1-1 with me:
+As our department continues to grow, I want to regularly check in with our teammates to keep a pulse on what is happening.
 
-1. Find a time on my calendar that is open and send me a calendar invite.
-2. Include as much context as is relevant so that I can prepare for the 1-1 if necessary.
+These 1-1s are intended to be casual and informal conversations to build deeper relationships, for me to listen, gather your feedback and insights, and discuss anything top of mind for you. These are not intended for status updates.
+
+If you receive a calendar invite from me but you need to change the date/time or you prefer not to meet, please update the invite.
```
Comment (nicksnyder) [inline content/company/team/nick.md]:
```suggestion
If you receive a calendar invite from me but you need to change the date/time or you prefer not to meet, please just let me know or update the invite directly.
```

File: content/company/team/nick.md [hunk 4]
```diff
@@ -18,10 +18,13 @@ If I am your new manager, I will setup a series of three extra 1-1s to learn mor
...
-If you would like to have a skip level 1-1 with me:
+As our department continues to grow, I want to regularly check in with our teammates to keep a pulse on what is happening.
 
-1. Find a time on my calendar that is open and send me a calendar invite.
-2. Include as much context as is relevant so that I can prepare for the 1-1 if necessary.
+These 1-1s are intended to be casual and informal conversations to build deeper relationships, for me to listen, gather your feedback and insights, and discuss anything top of mind for you. These are not intended for status updates.
+
+If you receive a calendar invite from me but you need to change the date/time or you prefer not to meet, please just let me know or update the invite directly.
+
+If you want to meet more frequently, please schedule time on my calendar or reach out to [Michal](index.md#michal-sennett) to help schedule.
```
Comment (nicksnyder) [inline content/company/team/nick.md]:
```suggestion
If you want to schedule so time to chat, feel free to send a calendar invite or reach out to [Michal](index.md#michal-sennett) to help schedule.

If you receive a calendar invite from me but you need to change the date/time or you prefer not to meet, you can update the invite directly or let [Michal](index.md#michal-sennett) know.
```

**Review by nicksnyder — APPROVED** [review]

**Pull request merged**